numpy/compat/_inspect.py

```diff
@@ -60,6 +61,35 @@ def iscode(object):
     """
     return isinstance(object, types.CodeType)
 
+def cleandoc(doc):
+    """Clean up indentation from docstrings.
+
+    Any whitespace that can be uniformly removed from the second line
+    onwards is removed."""
```

Comment:
Hmm, `can be uniformly removed` isn't very informative as to which whitespace is removed. Also, the closing `"""` should follow a blank line.